colors/nord.vim

```diff
@@ -572,7 +572,9 @@ call s:hi("LSPDiagnosticsWarning", s:nord13_gui, "", s:nord13_term, "", "", "")
 call s:hi("LSPDiagnosticsError" , s:nord11_gui, "", s:nord11_term, "", "", "")
 call s:hi("LSPDiagnosticsInformation" , s:nord8_gui, "", s:nord8_term, "", "", "")
 call s:hi("LSPDiagnosticsHint" , s:nord10_gui, "", s:nord10_term, "", "", "")
-
+hi! link LSPReferenceText CursorColumn
+hi! link LSPReferenceRead LSPReferenceText
+hi! link LSPReferenceWrite LSPReferenceText
```

Comment:
I guess this was added because this PR is created from the default (`develop`) branch instead of a story branch so changed to the fork are reflected here. Should be removed and maybe added in a new PR when it was intended to be included.

```suggestion
```